weedcoco/importers/voc.py

```diff
@@ -0,0 +1,145 @@
+from collections import defaultdict
+from pathlib import Path
+from typing import Optional, Mapping
+import argparse
+import json
+import yaml
+
+from lxml import etree
+
+from weedcoco.validation import validate
+
+
+def generate_coco_annotations(
+    voc_objects: list, image_id: int, start_id: int, category_mapping: Mapping[str, int]
+):
+    for annotation_id, voc_object in enumerate(voc_objects, start_id):
+        # TODO: handle segmented
+        annotation = {
+            "id": annotation_id,
+            "image_id": image_id,
+            "category_id": category_mapping[voc_object.find("name").text],
+            # TODO: handle truncated, difficult?
```

Comment:
What do you mean by truncated annotation?

Reply:
VOC allows an annotation to be tagged as "truncated". I think that means "only part of the object being labelled is visible in the image; it is cut off by the edge of the picture". I don't know what "difficult" is suppose to mean.

Reply:
From http://host.robots.ox.ac.uk/pascal/VOC/voc2006/htmldoc/index.html

![image](https://user-images.githubusercontent.com/78827/94085612-f5ea5980-fe4b-11ea-8ebc-357d9b5e99fd.png)

Reply:
Huh I wonder if people would consider our rye grass "difficult"...

Reply:
I think "difficult" here means "so difficult it's not worth demeriting a system for failing it"